pkg/model/dev.go

```diff
@@ -93,6 +89,13 @@ type Dev struct {
 	NodeSelector         map[string]string     `json:"nodeSelector,omitempty" yaml:"nodeSelector,omitempty"`
 	Affinity             *Affinity             `json:"affinity,omitempty" yaml:"affinity,omitempty"`
 	Metadata             *Metadata             `json:"metadata,omitempty" yaml:"metadata,omitempty"`
+
+	//Deprecated fields
```

Comment:
What does deprecated mean from a process standpoint? Is there an issue to remove these at some point in time. Do we need tests that assert things don't break if deprecated elements are gone?

Reply:
@jLopezbarb why are we deprecating these fields?